src/packet/null.rs

```diff
@@ -0,0 +1,37 @@
+use super::{CodecExtra, Depacketizer, MediaKind, PacketError, Packetizer};
+
+#[derive(Debug)]
+pub struct NullPacketizer;
+
+#[derive(Debug)]
+pub struct NullDepacketizer;
+
+impl Packetizer for NullPacketizer {
+    fn packetize(&mut self, _mtu: usize, b: &[u8]) -> Result<Vec<Vec<u8>>, PacketError> {
+        Ok(vec![b.to_vec()])
+    }
+
+    fn is_marker(&mut self, data: &[u8], previous: Option<&[u8]>, last: bool) -> bool {
+        true
```

Comment:
should this ever be hit?

Reply:
No. It takes the marker from the incoming packet. I should make a code comment.